src/Components/Card/index.jsx

```diff
@@ -2,23 +2,29 @@ import axios from "axios";
 import React from "react";
 import "./styles.css";
 import { useNavigate } from "react-router-dom";
```

Comment:
Suggestion - All these files can be imported into a index.js and imported together. Code look's cleaner and easy to read.